plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -33,8 +33,33 @@
 #define FU_PXI_DEVICE_CMD_FW_OTA_DISCONNECT	0x29
 
```

Comment:
randomly you seem to have changed the files to be executable, e.g. mode 0644 to 0755 -- which isn't a good idea.